formula/src/main/java/com/instacart/formula/Effect.kt

```diff
@@ -0,0 +1,47 @@
+package com.instacart.formula
+
+/**
+ * Effect is a function returned within [Transition.Result] which will be executed by
+ * [FormulaRuntime] usually after [Formula] state changes are applied. This function is
```

Comment:
I think `usually` might be misleading here. Is your intention that usually new state is provided, and effects are *always* executed after new state application? It might be misread as "sometimes effects are executed before state changes application".

Reply:
Good point, updated